src/modules/uavcan/uavcan_servers.hpp

```diff
@@ -88,7 +95,7 @@ class UavcanServers
 	static constexpr unsigned QueuePoolSize =
 		(NumIfaces * uavcan::MemPoolBlockSize * MaxCanFramesPerTransfer);
 
-	static constexpr unsigned StackSize  = 3500;
+	static constexpr unsigned StackSize  = 6000;
```

Comment:
A drastic change is coming up: https://github.com/UAVCAN/libuavcan/compare/footprint_reduction, ETA 2 days.